site/content/docs/5.3/content/tables.md

```diff
@@ -926,9 +923,10 @@ Use SVG to display icons or thumbnails in your compact table data cell elements
             </div>
           </td>
           <td>
-            <svg xmlns="http://www.w3.org/2000/svg" width="1.875rem" height="1.875rem" class="me-1" viewBox="0 0 30 30" role="img" aria-labelledby="svg2">
-              <title id="svg2">Document</title>
-              <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#document"></use>
+            <svg xmlns="http://www.w3.org/2000/svg" width="1.875rem" height="1.875rem" viewBox="0 0 30 30" role="img" aria-labelledby="svg1" class="me-2">
+              <title id="svg1">Document</title>
+              <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#document">
+              </use>
```

Comment:
Should be one line above

Reply:
Good catch! Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2034/commits/3ce373a6b28e43f486009c4ae7799b5926db9b95. I let you approve the PR if it's good for you so that it can land in the v5.3.0.